driver/src/order_driver.rs

```diff
@@ -127,7 +146,7 @@ mod tests {
         contract.get_current_block_timestamp.given(()).will_return(Ok(U256::from(200)));
         contract.order_hash_for_slot.given(slot).will_return(Ok(orders.rolling_hash(0)));
         contract.get_current_state_root.given(()).will_return(Ok(state_hash));
-        contract.calculate_order_hash.given((slot, Any)).will_return(Ok(H256::zero()));
+        contract.calculate_order_hash.given((slot, Any)).will_return(Ok(H256::from("0x438d54b20a21fa0b2f8f176c86446d9db7067f6e68a1e58c22873544eb20d72c")));
```

Comment:
Does this `0x438d54b20a21fa0b2f8f176c86446d9db7067f6e68a1e58c22873544eb20d72c` have a special meaning? Is it the hash of an empty order set?

Reply:
Yeah should maybe not be the default but set in the expectations of each test case.

Reply:
Ahh, yes, I hadn't noticed that this was a test case!